agenta-cli/agenta/cli/main.py

```diff
@@ -80,7 +80,8 @@ def cli():
 @click.command()
 @click.option("--app_name", default="")
 @click.option("--backend_host", default="")
-def init(app_name: str, backend_host: str):
+@click.option("--organisation", default="")
```

Comment:
default should be None and not "", by the way same for backend_host and app_name. "" is not a null value in python. using `if not var_name` comes with lots of bugs (for instance if it is an empty list). It's better to be explicit `if var_name is None` and use None as a default value